pipeline-runner/R/handle_data.R

```diff
@@ -153,17 +153,17 @@ upload_matrix_to_s3 <- function(pipeline_config, experiment_id, data) {
   saveRDS(data, file = count_matrix)
 
   message("Uploading updated count matrix to S3 bucket ", pipeline_config$processed_bucket, " at key ", object_key, "...")
-  s3$put_object(
-    Bucket = pipeline_config$processed_bucket,
-    Key = object_key,
-    Body = count_matrix
-  )
+  put_object_in_s3_multipart(
+    pipeline_config,
+    bucket = pipeline_config$processed_bucket,
+    object = count_matrix,
```

Comment:
Agi did the same thing in one of his more recent PRs, I think it also got merged in. Will be best to pull master in to get these changes

Reply:
Will do